src/main/java/name/webdizz/sonar/grammar/spellcheck/GrammarDictionaryLoader.java

```diff
@@ -42,4 +45,22 @@ public SpellDictionary loadMainDictionary() {
             locker.unlock();
         }
     }
+
+    Optional<SpellDictionaryHashMap> loadAlternateDictionary() {
+        String alternateDict = settings.getString(PluginParameter.ALTERNATIVE_DICTIONARY_PROPERTY_KEY);
+        Reader reader;
+        Optional<SpellDictionaryHashMap> result = Optional.absent();
+        if (alternateDict != null) {
+            alternateDict = alternateDict.replace(PluginParameter.SEPARATOR_CHAR, "\n");
+            reader = new StringReader(alternateDict);
+            try {
+                result = Optional.of(new SpellDictionaryHashMap(reader));
+            } catch (IOException e) {
+                throw new UnableToLoadDictionary("There is no file with dictionary.", e);

```

Comment:
just rephrase an exception message to not to confuse with file as we do not have file here, for example, Cannot load dictionary from SonarQube storage